### ISSUE TITLE
Make European swordsmen a little bit more resistant to crossbow bolts...

### ISSUE THREAD

**Issue by PitchNeeded** [Sh0wdown/UnofficialCrusaderPatch#275]
After the Arabian swordsmen were made more resistant to crossbow bolts near the start of this patch, I think European swordsmen should be made a little bit more resistant too to crossbow blots to balance things, especially because they are slower.

But only a small amount more resistant I think.

**Comment by ByBurton** [comment]
Why? They are already so much stronger than their arabian counterparts. And vs. crossbows you have pikemen as european lord.

**Comment by PitchNeeded** [comment]
> Why? They are already so much stronger than their arabian counterparts. And vs. crossbows you have pikemen as european lord.

Well I noticed in AI matches Frederick or Marshall's swordsmen struggle to reach the walls of Wolf or Pig because of all the crossbowmen. And in human vs AI matches, Frederick and Marshall have the same problem against a human player if the human player has a decent amount of crossbowmen (say about 30-40 crossbowmen at least) on towers at the front of their castle. I would say that a similar number of the slightly faster Arabian swordsmen often make slightly more progress now against crossbow fire than European swordsmen on their own without pikemen. I like the change to Arabian swordsmen and it should stay I think, but it just feels like now European swordsmen have been left behind slightly.

Another possibility would be to make European swordsmen very slightly faster, so they are still slower than Arabian swordsmen but not so pitifully slow as they are at the moment.

**Comment by Zer0oZ** [comment]
I disagree, european swordsmen are slower than their arabian counterparts because they're stronger and better armoured. The arabian swordsmen are a bit tougher now but they're still weaker and die alot faster than the european ones.

I agree that melee troops get mowed down by missile troops but this isn't a problem that concerns the troops themselves but the siege behavior of the AI. The AI should stay put and just use artillery to knock down towers before moving their melee troops forward. The amount of siege equipment should increase just like troop numbers to raise the effectiveness of AI sieges, the troop health is fine as it is in my opinion.

Imagine the AI using 5-7 trebuchets with shields infront besieging your castle (while being out of range of your missile troops of course), you'd have to sally out with knights to disable the artillery, thus forcing the AI to advance with regular troops. This would create a far more interesting and challenging experience as you couldn't just camp on towers with ballistas, rushing the walls with melee troops shouldn't work, artillery exists for a reason and the AI should use it alot more.

**Comment by PitchNeeded** [comment]
> Imagine the AI using 5-7 trebuchets with shields infront besieging your castle (while being out of range of your missile troops of course), you'd have to sally out with knights to disable the artillery, thus forcing the AI to advance with regular troops. This would create a far more interesting and challenging experience as you couldn't just camp on towers with ballistas, rushing the walls with melee troops shouldn't work, artillery exists for a reason and the AI should use it alot more.

The problem is most AIs verses other AIs would struggle with defending against this. Apart from maybe Marshall and Emir who are good at sending big groups of strong melee units out to counterattack, but even they don't always target enemy catapults/trebuchets with counter-attacks.

**Comment by Heroesflorian** [comment]
Right now, AI sieges are a bit underpowered though, especially with the doubling of defensive siege engines, more defensive troops (probably being extended with next couple releases), healers and proper replacing of killed defenders. 
All that offense gained (apart from sending their troops at the walls in bigger groups / less time in between) is going in properly once open, and an optional faster increase of attack army size (which also equals more "defenders" gathering on the keep for next attack, though). 
Upping siege engine amount - doubling them on par with defensive siege engine doubling would not be OP imho - would be a logical step for me. Although not just simply doubling existing siege engines but rather adding some shields or siege towers to the existing ones instead might possibly be more beneficial and interesting. 

Anyway, a maxed Wolf defense pretty much mows down about anything that enters its range already in vanilla, so with better AIVs and the patch improvements it's obvious that this won't get any better for a simple column of melee troops walking to the walls. European swordsmen still outperform anyone else except pikemen, though, so it is, as others already pointed out, not the swordsmen that are too weak - it is a general "melee troops walking towards heavily fortified castles to hug the walls" strategy issue. 
Will stronger sieges enable strong attackers to finish weak defenders faster? Probably, but so what? 
Currently, most castles will only fall to economic bankruptcy, or because of a severe lack of crossbowmen and own melee troops (e.g. when a Rat or Snake is overrun in Lategame by swordsman armies due to sheer troop quality differences).

**Comment by ByBurton** [comment]
@Heroesflorian You don't have to activate those aic files, or you can make some that lower the defence for AI lords like The Wolf, if you want. We improved the defence, yes, but everything is selectable.
If you think healers are too strong, disable them. 
The way AI reinforces their defences will be individual for each ai in the future (like it is in the original), it was only a test in the previous version.
If you only want stronger attacks, make them stronger, and let the defence the way it is.

**Comment by Heroesflorian** [comment]
I don't want to nerf the Wolf, I just wanted to share my opinion on the previous comment arguing stronger attacks would be OP.

**Comment by DarkThief-Darek** [comment]
I agree with Heroesflorian that the "melee troops walking towards heavily fortified castles to hug the walls" is the huge problem in the Stronghold series. The AIs are too simple to handle complex castle assault strategies right now. The only way the AI can siege somewhat successful is when they add ranged troops in the attack as well as spam siege equipment. Both of those things are really lacking in the Crusader castle attacks (too few siege equipment that gets shot down too easily before being built and the archers often times don't attack with the melee troops).

Besides the crossbowmen, who are too strong against every unit without any drawbacks I find the fire mechanic the worst for the current AI strategies. The attacking AI for example can't deal with a single fire thrower on a wall who will burn all troops, independent of quality or quantity. And we don't need to talk about pitch ditches, they are the worst (especially in front of walls). Oil engineers are bugged and have limited use, so they aren't such a problem right now.

On Topic: Maybe buff swordsman to crossbows a little bit, but also make them less resistance against archers to balance things out. I find it frustrating that a snake with a massive archer spam can do nothing against two swordsman slashing against the wall.

But I guess we should also focus on the speed of the swordsmen (both Europe and Arabian)
I think the speed of the pikemen are fine, but we shouldn't go slower than that, because slow units in strategy games are always heavily vulnerable to kiting, especially if they are so vulnerable to certain ranged attacks or environmental damage like fire. 
Think about the player too. I really hate using swordsman, because they are So! Slow! 
But I would want to use them if they were more effective. But pikemen or even macemen are so much more valuable with the current balance.
